Python/Product/PythonTools/PythonTools/Debugger/DebugAdapterLauncher.cs

```diff
@@ -45,12 +46,23 @@ public sealed class DebugAdapterLauncher : IAdapterLauncher {
             }
             return DebugAdapterProcess.Start(launchInfo.LaunchJson);
         }
+
         public void UpdateLaunchOptions(IAdapterLaunchInfo launchInfo) {
             if(launchInfo.LaunchType == LaunchType.Attach) {
                 launchInfo.DebugPort.GetPortName(out string uri);
-                JObject obj = new JObject {
-                    ["remote"] = uri
+
+                var shell = (IVsShell)Package.GetGlobalService(typeof(SVsShell));
```

Comment:
add comment:

// Ensure that package is loaded, as it is not guaranteed to be.
// Example: Start VS, Debug/Attach without first loading any Python file/project